src/applications/disability-benefits/526EZ/constants.js

```diff
@@ -84,3 +86,12 @@ export const ADDRESS_TYPES = {
   mailingAddress: 'mailingAddress',
   forwardingAddress: 'forwardingAddress'
 };
+
+export const itfStatuses = new Proxy({
+  noneFound: 'none found',
```

Comment:
is this meant to be snake cased?

Reply:
Oh, this should actually be deleted; it's not returned from the API and we'll know if we have no ITFs by whether the `intentToFile` array is empty.